JSDemos/Demos/Toolbar/Multiline/Angular/app/app.component.html

```diff
@@ -0,0 +1,187 @@
+      ></dxi-item>
+
+      <dxi-item
+        location="before"
+        locateInMenu="never"
+        widget="dxButtonGroup"
+        [options]="fontStyleOptions"
+      ></dxi-item>
+
+      <dxi-item
+        location="before"
+        locateInMenu="never"
+        template="<div class='separator'></div>"
+      ></dxi-item>
+
+      <dxi-item
+        cssClass="text-align-group"
+        location="before"
+        locateInMenu="auto"
+        widget="dxButtonGroup"
+        [options]="textAlignOptions"
+      ></dxi-item>
+
+      <dxi-item
+        location="before"
+        locateInMenu="never"
+        widget="dxButtonGroup"
+        displayExpr="text"
+        [options]="listOptions"
+      ></dxi-item>
+
+      <dxi-item
+        location="before"
+        locateInMenu="auto"
+        template="<div class='separator'></div>"
+        menuItemTemplate="<div class='menu-separator'></div>"
+      ></dxi-item>
+
+      <dxi-item
+        location="before"
+        locateInMenu="auto"
+        widget="dxDateBox"
+        [options]="dateBoxOptions"
+      ></dxi-item>
+
+      <dxi-item
+        location="before"
+        locateInMenu="auto"
+        template="<div class='separator'></div>"
+        menuItemTemplate="<div class='menu-separator'></div>"
+      ></dxi-item>
+
+      <dxi-item
+        location="before"
+        locateInMenu="auto"
+        widget="dxCheckBox"
+        showText="inMenu"
+        [options]="checkBoxOptions"
+      ></dxi-item>
+
+      <dxi-item
+        location="after"
+        locateInMenu="never"
+        widget="dxButton"
+        showText="inMenu"
+        [options]="attachOptions"
+      ></dxi-item>
+
+      <dxi-item
+        location="after"
+        locateInMenu="auto"
+        widget="dxButton"
+        showText="inMenu"
+        [options]="addButtonOptions"
+      ></dxi-item>
+
+      <dxi-item
+        location="after"
+        locateInMenu="auto"
+        widget="dxButton"
+        showText="inMenu"
+        [options]="removeButtonOptions"
+      ></dxi-item>
+
+      <dxi-item
+        location="after"
+        locateInMenu="auto"
+        widget="dxButton"
+        showText="inMenu"
+        [options]="aboutButtonOptions"
+      ></dxi-item>
+
+      <div *dxTemplate="let data of 'colorpicker'">
+        <div class="custom-color-picker">
+          <i
+            *ngFor="let iconColor of colors"
+            class="color dx-icon dx-icon-square"
+            [ngClass]="{ 'dx-theme-text-color': !iconColor }"
+            [style.color]="iconColor"
+            (click)="onColorClick(iconColor)"
+          ></i>
+        </div>
+      </div>
+
+      <div *dxTemplate="let data of 'fontSizeTemplate'">
+        <div [style.fontSize]="data.size + 'px'">{{ data.text }}</div>
+      </div>
+    </dx-toolbar>
+  </dx-resizable>
+
+  <div class="options-container">
+    <div class="caption">Options</div>
+
+    <dx-radio-group
+      layout="horizontal"
+      [items]="priorities"
```

Comment:
Why is it names as `priority`? I think it should be something like `toolbarLineMode`

Reply:
https://github.com/DevExpress/devextreme-demos/pull/2004/commits/5ba43954aa5e53c1a0c2a8596281ef2bf941d78c